pom.xml

```diff
@@ -87,7 +87,7 @@
 
         <!-- core lib dependency -->
         <bytebuddy.version>1.12.13</bytebuddy.version>
-        <grpc.version>1.44.0</grpc.version>
+        <grpc.version>1.50.0</grpc.version>
```

Comment:
`protobuf` should also be updated to be aligned with `grpc`

Reply:
grpc have fix protobuf

Reply:
If it does, you should consider bumping up gRPC as well.

Reply:
yes,I find gRPC vesion config use same configuration。 All update to 1.50.0

Reply:
`com.google.protobuf.protoc.version` and `protoc-gen-grpc-java.plugin.version` should be updated as well.

And please check the netty version grpc-java bundling, which our 4.1.79.Final is suitable or lower.

Reply:
update it. check the netty version is ok.